source/common/runtime/runtime_features.cc

```diff
@@ -47,6 +47,12 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    "envoy.deprecated_features.bootstrap.proto:runtime",
+    "envoy.deprecated_features.redis_proxy.proto:catch_all_cluster",
+    "envoy.deprecated_features.lds.proto:use_original_dst",
+    "envoy.deprecated_features.server_info.proto:max_stats",
```

Comment:
I think this and below are on output protos so they won't have any effect. Not sure how we could know this. Thoughts?

Reply:
Good point.  No ideas here - I'd be inclined to leave them in to avoid script churn.
Maybe have file or proto naming in v3 to make things more clear going forward?
@htuch